tests/.setup.js

```diff
@@ -1,17 +1,8 @@
-const jsdom = require('jsdom');
```

Comment:
This commit has the bulk of the changes. The test files aren't perfect but good enough to move us forward. In the future we can try to improve the coverage.

Reply:
Yeah, I feel like these tests are really not enough. But this is something for the future.